src/easylink/step.py

```diff
@@ -1210,6 +1226,92 @@ def _validate(self) -> None:
         if errors:
             raise ValueError("\n".join(errors))
 
+    def validate_step(
+        self,
+        step_config: LayeredConfigTree,
+        combined_implementations: LayeredConfigTree,
+        input_data_config: LayeredConfigTree,
+    ) -> dict[str, list[str]]:
+        """Validates the ``Step`` assigned to this ``EmbarrassinglyParallelStep``.
+
+        Parameters
+        ----------
+        step_config
+            The internal configuration of this ``Step``, i.e. it should not include
+            the ``Step's`` name.
+        combined_implementations
+            The configuration for any implementations to be combined.
+        input_data_config
+            The input data configuration for the entire pipeline.
+
+        Returns
+        -------
+            A dictionary of errors, where the keys are the ``Step`` name and the
+            values are lists of associated error messages.
+
+        Notes
+        -----
+        If the ``EmbarrassinglyParallelStep`` does not validate (i.e. errors are
+        found and the returned dictionary is non-empty), the tool will exit and
+        the pipeline will not run.
+
+        We attempt to batch error messages as much as possible, but there may be
+        times where the configuration is so ill-formed that we are unable to handle
+        all issue in one pass. In these cases, new errors may be found after the
+        initial ones are handled.
+        """
+        return self.step.validate_step(
+            LayeredConfigTree(step_config),
+            combined_implementations,
+            input_data_config,
+        )
```

Comment:
I feel this could use a comment, as I'm puzzling through it a bit. By passing the entire `step_config`, you are essentially saying there is no `config_key` for this step type? Have I got that right?

Reply:
This isn't actually really needed quite yet and I can add it in the next PR. Ultimately, though, I think we're going to need to validate the sub-step much like we [validate the TemplatedStep's template_step](https://github.com/ihmeuw/easylink/blob/main/src/easylink/step.py#L812-L814).

(and I don't actually need to wrap it in LayeredConfigTree - it should already be one).

And yes, the implication by not subsetting the step_config is that it should never have a config key at this point (which I believe to be true?)

I'll remove this entire function for now and we can discuss if needed in the actual implementation PR.